plugins/modules/vmware_dvs_portgroup.py

```diff
@@ -3,6 +3,7 @@
 
 # Copyright: (c) 2015, Joseph Callen <jcallen () csc.com>
 # Copyright: (c) 2017, Ansible Project
+# Copyright: (c) 2018,2020 Christian Kotte (@ckotte)
```

Comment:
```suggestion
# Copyright: (c) 2018, Christian Kotte (@ckotte)
```

In one of the Ansible Doc meeting, it was decided to keep the first year of commit in copyright